android/gradle.properties

```
@@ -0,0 +1,4 @@
+AppleAuth_kotlinVersion=1.3.50
+AppleAuth_compileSdkVersion=28
+AppleAuth_buildToolsVersion=28.0.3
+AppleAuth_targetSdkVersion=28
```

Why not Kotlin current? Or if 1.4.0 is too crazy, 1.3.70?
Android needs to go to API29 to be used in new apps already (as of August) and for updates as of Nov

```suggestion
AppleAuth_kotlinVersion=1.4.0
AppleAuth_compileSdkVersion=29
AppleAuth_buildToolsVersion=29.0.3
AppleAuth_targetSdkVersion=29
```